test/integration/react-rsc-basic/test/css.test.js

```diff
@@ -0,0 +1,94 @@
+/* eslint-env jest */
```

Comment:
you can make this file as css.js and import it in index.test.js to run, aliging with other integration test